package.json

```diff
@@ -109,7 +109,7 @@
     "@bbc/psammead-styles": "7.3.0",
     "@bbc/psammead-timestamp": "4.0.19",
     "@bbc/psammead-timestamp-container": "5.0.30",
-    "@bbc/psammead-topic-tags": "0.1.0-alpha.10",
+    "@bbc/psammead-topic-tags": "0.1.0-alpha.11",
```

Comment:
I doubt we should be deploying an alpha component to live? Should this be released as version 1.0.0 and integrated in this PR?

Reply:
Yep just about to set up that PR 👍